modules/op-release-notes-1-10.adoc

```diff
@@ -0,0 +1,175 @@
+
+[id="new-features-1-10_{context}"]
+== New features
+
+In addition to the fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.10.
+
+[id="pipelines-new-features-1-10_{context}"]
+=== Pipelines
+
+* With this update, you can specify environment variables in a `PipelineRun` or `TaskRun` pod template to override or append the variables that are configured in a task or step. Also, you can specify environment variables in a default pod template to use those variables globally for all `PipelineRuns` and `TaskRuns`. This update also adds a new default configuration `forbidden-ends` to filter the environment variables while propagating from pod templates.
+* With this update, for a cluster resolver, bundle resolver, and Git resolver, a pipeline sets the `ConfigSource` value to the source from where the remote resources were received. The `ConfigSource` parameter includes a source URL and hex-encoded sha256 checksum of the cluster resource.
+* This update adds a new boolean flag `enable-provenance-in-status` into the `feature-flags` config map to enable the provenance field in status to be populated. The provenance field records authenticated metadata about how a software artifact was built.
+* This update adds reconciler, event, config, and webhook support for a custom run.
+* This update populates the `Status.Provenance.ConfigSource` parameter for a `TaskRun` or `PipelineRun` by using the value from the remote `ResolutionRequest` status.
++
+[NOTE]
+====
+You need to set the feature flag `enable-provenance-in-status` to `true` to enable the `Status.Provenance.ConfigSource` parameter to be populated and available in a run status.
+====
+
+* With this update, custom tasks in pipelines are enabled by default. 
++
+[NOTE]
+====
+To disable this update, set the `enable-custom-tasks` flag to `false` in the `feature-flags` config map file.
+====
+
+* This update adds support for the `PipelineRun` reconciler to create a custom run. For example, custom `TaskRuns` created from `PipelineRuns` can now use API version `v1beta1.CustomRuns` instead of `v1alpha1.Runs`, if the `custom-task-version` feature flag is set to `v1beta1`, instead of the default value `v1alpha1`. 
++
+[NOTE]
+====
+You need to update the custom task controller to listen for the `*v1beta1.CustomRun` API version instead of `*v1alpha1.Run` in order to respond to `v1beta1.CustomRun` requests.
+====
+
+* This update adds a new parameter `Retries` into `v1beta1.TaskRun` and `v1.TaskRun` API versions.
+* This update includes the following changes in the trusted resources verification package: 
+** Enable Key Management Service (KMS) in the `VerificationPolicy` library for trusted resources.
+** Add a new parameter `KMS` into `v1alpha1.VerificationPolicy` API version. Use this parameter to configure the KMS path and resolve the stored keys.
+** Add KMS libraries to use KMS for verification purposes.
```

Comment:
This is alpha, probably not worth having it there yet.

Reply:
@vdemeester Do you mean i have to remove only this point `Add KMS libraries to use KMS for verification purposes` or the entire bullet point `This update includes the following changes in the trusted resources verification package: `.

Reply:
Removed the entire bullet point